jenkins.io.tf

```diff
@@ -0,0 +1,41 @@
+resource "azurerm_resource_group" "jenkins_io" {
+  name     = "jenkinsio"
+  location = var.location
+}
+
+resource "azurerm_storage_account" "jenkins_io" {
+  name                              = "jenkinsio"
+  resource_group_name               = azurerm_resource_group.jenkins_io.name
+  location                          = azurerm_resource_group.jenkins_io.location
+  account_tier                      = "Premium"
+  account_kind                      = "FileStorage"
+  access_tier                       = "Hot"
+  account_replication_type          = "ZRS"
+  min_tls_version                   = "TLS1_2" # default value, needed for tfsec
+  infrastructure_encryption_enabled = true
+
+  # Adding a network rule with `public_network_access_enabled` set to `true` (default) selects the option "Enabled from selected virtual networks and IP addresses"
+  network_rules {
+    default_action = "Deny"
+    ip_rules = flatten(
+      concat(
+        [for key, value in module.jenkins_infra_shared_data.admin_public_ips : value],
+      )
+    )
+    virtual_network_subnet_ids = [
+      data.azurerm_subnet.publick8s_tier.id,
+      data.azurerm_subnet.privatek8s_tier.id,
```

Comment:
Do we write anything on this SAS from infra.ci.jenkins.io or release.ci.jenkins.io?

If yes you can dismiss this suggestion. Otherwise let's remove this line as per the least principle privilege

```suggestion
```